packages/main/src/StandardListItem.hbs

```diff
@@ -7,8 +7,8 @@
 			<div class="ui5-li-description-info-wrapper">
 				<span part="description" class="ui5-li-desc">{{description}}</span>
 
-				{{#if info}}
-					<span part="info" class="ui5-li-info">{{info}}</span>
+				{{#if additionalText}}
+					<span part="additionalText" class="ui5-li-info">{{additionalText}}</span>
```

Comment:
I'm not familiar with part but should we keep it value in camelCase style?

Reply:
@nnaydenow we decided to write the CSS Shadow Parts in kebab-case

Reply:
@ilhan007 we can submit this when @fifoosid fix this